plugins/lang/statsx.txt

```diff
@@ -63,17 +63,17 @@ ATTACKERS = Angreifer
 ACC = Genauigkeit
 HIT_S = Treffer
 DMG = Schaden
-VICTIMS = Sieger
+VICTIMS = Opfer
 MOST_DMG = Am meisten verursachter Schaden durch
-KILLED_YOU_DIST = %s hat dich getoetet ^naus einer Entfernung von %0.2f Metern.
+KILLED_YOU_DIST = %s hat dich mit einer %s ^naus %0.2f Metern Entfernung getoetet.
 DID_DMG_HITS = Er verursachte %d Schaden bei dir mit %d Treffern^nund hat noch %dhp und %dap.
-YOU_DID_DMG = Du hast %d Schaden bei ihm angerichtet mit %d Treffern.
+YOU_DID_DMG = Du hast bei ihm %d Schaden angerichtet mit %d Treffern.
 EFF = Effizienz
 BEST_SCORE = Die meisten Punkte
 KILL_S = Kill(s)
 TOTAL = Total
 SHOT_S = Schuesse
-HITS_YOU_IN = %s Treffer bei ihm
+HITS_YOU_IN = %ss Treffer bei dir
```

Comment:
This could also be a (simpler) direct translation from the original: "%s trifft dich in". Would also avoid the awkward "%s's" situation.